pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -1,73 +1,76 @@
+## Prerequisites
+You need a running, non-validating full node that is connected to network. 
 
-```bash
-dydxprotocold config chain-id $CHAIN_ID
-```
-
-### Getting a Snapshot
+- If you created a system service for your node by following the instructions on the previous page, [Set Up a Full Node](../infrastructure_providers-validators/how_to_set_up_full_node.md), start your node with the following command:
+  ```bash
+  stystemctl start dydxprotocol
+  ```
+- To start your node directly with Cosmovisor, you must include the flag `--non-validating-full-node=true`. Cosmovisor may prompt you to configure additional environment variables or include them in your command.
+  ```bash
+  cosmovisor run start --non-validating-full-node=true 
+  ```
 
-See [snapshot service](../infrastructure_providers-network/resources.mdx#snapshot-service).
+<!-- is this right? i also see 
+dydxprotocold start --p2p.seeds="..." --bridge-daemon-eth-rpc-endpoint="<eth rpc endpoint>" --non-validating-full-node=true -->
 
-### Starting a Full Node
+## Connect to Healthy Peers
+To keep your full node up to date, connect to healthy peers that have the latest state of the network. You can find a list of healthy peers with the latest state from the following services:
+- https://services.lavenderfive.com/mainnet/dydx#live-peers
+- https://polkachu.com/live_peers/dydx
 
-Find the seed node's ID and the IP address from [Resources](../infrastructure_providers-network/resources.mdx#seed-nodes). Then, run the following command to start a non-validating full node.
+From a list of healthy peers, choose any 5 for your node to query for the latest state. Add a comma-separated list of those peer addresses to the `persistent_peers` field in your `config.toml`, like in the following example:
 
-For example,
-```bash
-dydxprotocold start --p2p.seeds="..." --bridge-daemon-eth-rpc-endpoint="<eth rpc endpoint>" --non-validating-full-node=true
+```yaml
+# config.toml
+# Example values from Polkachu for dydx-mainnet-1
+persistent_peers=83c299de2052db247f08422b6592e1383dd7a104@136.243.36.60:23856,1c64b35055d34ff3dd199bb4a5a3ae46b9c10c89@3.114.126.71:26656,3651c82a89f8f4d6fc30fb27b91159f0de092031@202.8.9.134:26656,580ec248de1f41d4e50abe132b7838348db55b80@176.9.144.40:23856,febe75fb6e70a60ce6344b82ff14903bcb53a209@38.122.229.90:26656
 ```
 
-💡**Note**: if you want to disable gRPC on your full node, it is important to start the node with the
-`--non-validating-full-node=true` flag. Otherwise, the application will require that gRPC be enabled.
-
-
-## Staying up-to-date with the Latest State
-
-### Connecting to Healthy Peers
-
-In order for the full node to have the latest state, it needs to connect to peers which have the latest state Use the following links to get a list of live peers which have the latest state
-
-https://services.lavenderfive.com/mainnet/dydx#live-peers
+<!-- do these need to be in square brackets? it seems like thats how toml arrays work -->
```

Comment:
I can't tell if this works yet. testing more, curious if its supposed to be in brackets to denote a toml array

Reply:
They dont need to be IIRC

Reply:
looks like you're right